<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="netTab">
        <attribute name="title">
         <string>Network View</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTreeWidget" name="treeWidget">
           <column>
            <property name="text">
             <string>Tree View</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>Networks</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Packets</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Messages</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="packetTab">
        <attribute name="title">
         <string>Packets View</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelView">
          <property name="text">
           <string>Slect from tree</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProject"/>
   <addaction name="actionToggleFilter"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewProject">
   <property name="text">
    <string>NewProject</string>
   </property>
  </action>
  <action name="insertRowAction">
   <property name="text">
    <string>insertRow</string>
   </property>
  </action>
  <action name="deleteRowAction">
   <property name="text">
    <string>deleteRow</string>
   </property>
  </action>
  <action name="actionToggleFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ToggleFilter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
